### DISCUSSION TITLE
addon settings: placeholder instead of value

### DISCUSSION THREAD

**Discussion by cobaltt7** [ScratchAddons/ScratchAddons#1432] [Help] [answered]
when I make my addon setting's id "followuser", it starts making the default value a placholder instead of a value! why does it do this?

**Comment by WorldLanguages** [comment, marked as answer]
Keep in mind defaults are fallbacks - you need to delete your Scratch Addons settings to check whether defaults are being put properly. Try running `chrome.storage.sync.clear()` in the settings page console, then reload the extension.
In settings of type text, defaults are also used as placeholders.

**Reply by cobaltt7** [reply]
ohh i understand now!  i must have set that setting to blank before i set the default value. then, the default value wasn't showing because the setting was already set to "false". a setting's default value is also a placeholder, so i was still seeing it since the field was blank.

thanks for explaining!
